lib/active_shipping/carriers/ups.rb

```diff
@@ -626,7 +626,9 @@ def response_success?(document)
     end
 
     def response_message(document)
-      document.root.at_xpath('Response/Error/ErrorDescription | Response/ResponseStatusDescription').text
+      status = document.root.at_xpath('Response/ResponseStatusDescription').try(:text)
+      desc = document.root.at_xpath('Response/Error/ErrorDescription').try(:text)
+      [status, desc].select(&:present?).join(": ").presence || "UPS could not process the request."
```

Comment:
Are there ever actually cases where we just get one of these? Status or description?